layouts/html.hbs

```diff
@@ -145,6 +145,10 @@
             HitchhikerJS.RuntimeConfigReceiver.handle(message.runtimeConfig);
           } else if (window.isOverlay) {
             window.Overlay.onMessage(message);
+          } else if (message.eventType && message.eventType === 'click') {
+            ANSWERS.components._activeComponents
```

Comment:
@oshi97 or @tmeyer2115 , I'm wondering how you all feel about using `ANSWERS.components._activeComponents` since it's a private property. Alternatively, we could add a public function to the SDK which would allow us to get all of autocomplete components